js/app.js

```diff
@@ -1,186 +1,84 @@
+var store1 = new Store ('First And Pike', 6.3,['','6am','7am','8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],23,65);
+render(store1);
+
+var store2 = new Store ('SeaTac Airport', 1.2,['','6am','7am','8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],3,24);
+render(store2);
+
+var store3 = new Store ('Seattle Center', 3.7,['','6am','7am','8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],11,38);
+render(store3);
 
-var stores = [store1,store2,store3,store4,store5];
-console.log(stores);
+var store4 = new Store ('Capital Hill', 2.3,['','6am','7am','8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],20,38);
+render(store4);
 
-function randomCustomers (min,max) {
-  return Math.floor(Math.random() * (max - min + 1) + min);
+var store5 = new Store ('Alki', 4.6,['','6am','7am','8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'],2,16);
+render(store5);
+
+
+function randomCustomers (minCookie,maxCookie) {
+  return Math.floor(Math.random() * (maxCookie - minCookie + 1) + minCookie);
 };
 
-var parentElement = document.getElementById('sales');
 
-var article = document.createElement('article');
-parentElement.append(article);
 
-//Store 1
-var h2 = document.createElement('h2');
-h2.textContent = store1.name;
-article.appendChild(h2);
 
-var p = document.createElement('p');
-p.textContent = 'Sales by hour for the ' + store1.name + ' store.';
-article.appendChild(p);
+for (var x = 0; x < store1.hoursOpen.length; x++) {
+  var th = document.createElement ('th');
+  th.textContent = store1.hoursOpen [x] ;
+  row.appendChild(th);
+};
 
-var ul = document.createElement('ul');
-article.appendChild(ul);
 
-var totalCookies = 0;
+article.append(table);
 
-for (var i = 0; i < store1.hoursOpen.length; i++) {
-  var li = document.createElement ('li');
-  li.textContent = store1.hoursOpen [i] + ': ' + store1.cookiesSold() + ' cookies';
-  (totalCookies += store1.cookiesSold());
-  ul.appendChild(li);
-}
-li.textContent = 'Total: ' + totalCookies + ' cookies';
+function render (Store) {
+  console.log(Store);
 
-//Store 2
-h2 = document.createElement('h2');
-h2.textContent = store2.name;
-article.appendChild(h2);
+  var storeRow = document.createElement('tr');
+  var td = document.createElement ('td');
 
-p = document.createElement('p');
-p.textContent = 'Sales by hour for the ' + store2.name + ' store.';
-article.appendChild(p);
+  td.textContent = Store.name;
+  storeRow.appendChild(td);
 
-ul = document.createElement('ul');
-article.appendChild(ul);
+  var totalCookies = 0;
+  for (var i = 0; i < Store.hoursOpen.length; i++) {
+    td = document.createElement ('td');
+    td.textContent = Store.cookiesSold();
+    (totalCookies += Store.cookiesSold());
+    storeRow.appendChild(td);
+    table.appendChild(storeRow);
+  }
+  td.textContent = 'Total: ' + totalCookies;
```

Comment:
It may look a little cleaner to add Total to your header and omit it from each row